packages/components/psammead-brand/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@bbc/psammead-brand",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "description": "Provides the BBC News logo (as SVG), nested a hardcoded link to https://www.bbc.co.uk/news",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-brand"
+  },
+  "author": {
+    "name": "Psammead Maintainers",
+    "email": "PsammeadMaintainers@bbc.co.uk"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-brand/README.md",
+  "dependencies": {
+    "@bbc/gel-constants": "^0.1.1",
+    "@bbc/gel-foundations-styled-components": "^0.1.0",
+    "@bbc/psammead-styles": "^0.1.3",
+    "@bbc/psammead-visually-hidden-text": "^0.1.0",
+    "styled-components": "^4.1.2"
+  },
+  "devDependencies": {
+    "@bbc/psammead-test-helpers": "^0.1.0",
+    "react": "^16.6.3"
```

Comment:
React is used as a regular dependancy so shouldnt be in devDependancies https://github.com/BBC-News/psammead/pull/105/files#diff-4f91a28dd0d1d26145e763893261c522R1

Reply:
@dr3 - `babel-plugin-styled-components` added for the wrong reason. I initially thought this was needed per https://www.styled-components.com/docs/tooling#babel-plugin. But when I pull it out I have no errors. `babel-plugin-styled-components` was a line of inqurity resolved through doing `import styled, { css } from 'styled-components';` in `src/index.jsx`.